spec/Vehicle/Vehicle.vspec

```diff
@@ -399,10 +399,48 @@ CurrentLocation.Altitude:
   datatype: double
   type: sensor
   unit: m
-  description: Current altitude relative to WGS 84 reference ellipsoid.
+  description: Current altitude relative to WGS 84 reference ellipsoid, as measured at position of GNSS receiver antenna.
 
 CurrentLocation.VerticalAccuracy:
   datatype: double
   type: sensor
   unit: m
   description: Accuracy of altitude.
+
+CurrentLocation.GNSSReceiver:
+  type: branch
+  description: Information on the GNSS receiver used for determining current location.
+
+CurrentLocation.GNSSReceiver.Status:
+  datatype: string
+  type: sensor
+  allowed: ['NO_FIX', 'FIX_2D', 'FIX_3D']
```

Comment:
SENSORIS is here also more specific, allowing for the use of GNSS augmentation systems (correction signals given from GNSS satellites or ground stations)